native/kzg_nif/src/lib.rs

```diff
@@ -0,0 +1,182 @@
+use std::io::Write;
+use std::path::Path;
+
+use rustler::{Binary, Env, NewBinary};
+
+pub use c_kzg::{Bytes32, Bytes48, Blob, KzgSettings, KzgCommitment, KzgProof, BYTES_PER_COMMITMENT, Error};
+
+pub(crate) fn bytes_to_binary<'env>(env: Env<'env>, bytes: &[u8]) -> Binary<'env> {
+    let mut binary = NewBinary::new(env, bytes.len());
+    // This cannot fail because bin size equals bytes len
+    binary.as_mut_slice().write_all(bytes).unwrap();
+    binary.into()
+}
+
+fn get_trusted_setup() -> KzgSettings {
+    let lib_path = std::env::current_dir().unwrap().join(file!());
+    let src_path = lib_path.parent().unwrap();
+    let main_path = src_path.parent().unwrap();
+    let trusted_setup_path = main_path.join("native/kzg_nif/official_trusted_setup.txt");
+    let trusted_setup_file: &Path = trusted_setup_path.as_path();
+    assert!(trusted_setup_file.exists());
```

Comment:
We should avoid adding this on release builds by using `debug_assert!`. Also, let's add a comment to make the error's source clearer.
```suggestion
    debug_assert!(trusted_setup_file.exists(), "Missing trusted setup file");
```